### ISSUE TITLE
Can't compile handlebars for URL scalar property from .gql schema

### ISSUE THREAD

**Issue by kirudenk** [anvilco/spectaql#262]
I have a custom scalar `URL` and a type that uses it:
```
"""
A field whose value conforms to the standard URL format as specified in RFC3986: https://www.ietf.org/rfc/rfc3986.txt.
"""
scalar URL

type Course {
...
url: URL,
...
}
```
When trying to generate docs for that schema using config.yml from the examples I get this error:
```
Running "compile-handlebars:compile" (compile-handlebars) task
Task error: [TypeError: this[kFormat] is not a function
  at Object.toJSON (internal/url.js:691:27)
  at JSON.stringify (<anonymous>:null:null)
  at _stringify (/Users/.../spectaql/node_modules/json-stringify-pretty-compact/index.js:34:19)
```

When I change course to:
```
type Course {
...
url: String,
...
}
```
everything works. Also I didn't have that issue with 0.15.0 version. Thanks!

**Comment by newhouse** [comment]
Should be fixed in `1.0.5`...

**Comment by kirudenk** [comment]
@newhouse Thanks!
